tasks/src/feature-extraction/about.md

```diff
@@ -0,0 +1,60 @@
+## Use Cases
+
+Feature extraction is the task of extracting meaningful data from the given dataset. It is instrumental in reducing redundant/garbage data. Feature extraction is one of the most trivial steps in machine learning and is used heavily in NLP. In simple terms, Feature Extraction is transforming textual data into numerical data.
+ 
+## Task Variants 
+The task of feature extraction can be divided into three different variants:
+
+### Countvectorizer 
+CountVectorizer is a feature extraction technique that converts a collection of text documents to a vector of term/token counts. It is also known as the bag-of-words model. This model is widely used in text mining and information retrieval. 
+### TF-IDF Vectorizer
+TF-IDF stands for Term Frequency-Inverse Document Frequency. It is a numerical statistic that echoes a word's importance to a document in a corpus. In order to account for the fact that some words are used more frequently than others, the TF-IDF value rises according to the number of times a word occurs in the text and is offset by the number of documents in the corpus that contain the term.
+
+### Word Embeddings
+Word embeddings are a type of word representation that allows words with similar meanings to have an equivalent representation. Each word is transformed into a real-valued vector in a lower-dimensional space. Word embedding preserves contexts and relationships of words so that it detects similar words more accurately.
+
+
+## Inference
+
+You can infer with feature extraction through the 🤗 Transformers library using the [`feature-extraction`](https://huggingface.co/docs/transformers/en/main_classes/pipelines#transformers.FeatureExtractionPipeline) pipeline. The pipeline returns a list of feature vectors, one for each input text.
+
+```python
+
+#importing the library 
```

Comment:
This kinda doesn't explain how it can be useful, as this pipeline doesn't have a model head and the numbers returned don't really mean anything unless you put a head on the model. Maybe demonstrating it would be cool?